recipes-bsp/opensbi/opensbi-beaglev_0.8.0.bb

```diff
@@ -0,0 +1,48 @@
+SUMMARY = "RISC-V Open Source Supervisor Binary Interface (OpenSBI)"
```

Comment:
I think this file should be called 'opensbi-starfive*'

Reply:
Also can we re-use the existing OpenSBI functions more here?

Reply:
Can look into this more once we have **something** working on the hardware first